app/src/main/java/com/imdbmovieapp/data/local/entity/FavoriteMovieEntity.kt

```diff
@@ -0,0 +1,15 @@
+package com.imdbmovieapp.data.local.entity
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "movieTable")
+data class FavoriteMovieEntity(
+    @PrimaryKey
+    val id: Int,
+    val title: String,
+    val year: String,
+    val genre: String,
+    val image: String,
+    val favoriteMovie: Int,
```

Comment:
I still don't think it's clear what favoriteMovie parameter is for. Naming should give you impression of the parameter's aim/usage